cmd/aries-agent-mobile/pkg/wrappers/command/aries.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage/mem"
+
+	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/api"
+	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/wrappers"
+	"github.com/hyperledger/aries-framework-go/pkg/controller"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/command"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries"
+)
+
+// Aries is an implementation of AriesController which handles requests locally.
+type Aries struct {
+	framework *aries.Aries
+	handlers  map[string]map[string]command.Exec
+}
+
+// NewAries returns a new Aries instance that contains handlers and an Aries framework instance.
+func NewAries() (*Aries, error) {
+	// TODO receive options
+	storageProvider := mem.NewProvider()
+
+	framework, err := aries.New(aries.WithStoreProvider(storageProvider))
+	if err != nil {
+		return nil, fmt.Errorf("failed to initialize Aries framework: %v", err)
```

Comment:
use %w when wrapping errors (instead of %v)

Reply:
thanks, i will use this instead. i didn't know about it actually